specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,9538 @@
+        }
+      ],
+      "properties": {
+        "kind": {
+          "enum": [
+            "AzureSubscriptionCredential",
+            "AzureSubscriptionMsi",
+            "AzureResourceGroupCredential",
+            "AzureResourceGroupMsi",
+            "AzureSynapseWorkspaceCredential",
+            "AzureSynapseWorkspaceMsi",
+            "AzureSynapseCredential",
+            "AzureSynapseMsi",
+            "AdlsGen1Credential",
+            "AdlsGen1Msi",
+            "AdlsGen2Credential",
+            "AdlsGen2Msi",
+            "AmazonAccountCredential",
+            "AmazonS3Credential",
+            "AmazonS3RoleARN",
+            "AmazonSqlCredential",
+            "AzureCosmosDbCredential",
+            "AzureDataExplorerCredential",
+            "AzureDataExplorerMsi",
+            "AzureFileServiceCredential",
+            "AzureSqlDatabaseCredential",
+            "AzureSqlDatabaseMsi",
+            "AzureSqlDatabaseNativeAuth",
+            "ArcEnabledSqlServerCredential",
+            "ArcEnabledSqlServerNativeAuth",
+            "AmazonPostgreSqlCredential",
+            "AzurePostgreSqlCredential",
+            "DatabricksCredential",
+            "SqlServerDatabaseCredential",
+            "AzureSqlDatabaseManagedInstanceCredential",
+            "AzureSqlDatabaseManagedInstanceMsi",
+            "AzureSqlDatabaseManagedInstanceNativeAuth",
+            "AzureSqlDataWarehouseCredential",
+            "AzureSqlDataWarehouseMsi",
+            "AzureMySqlCredential",
+            "HdfsCredential",
+            "AzureStorageCredential",
+            "AzureStorageMsi",
+            "TeradataTeradataCredential",
+            "TeradataTeradataUserPass",
+            "TeradataUserPass",
+            "OracleOracleCredential",
+            "OracleOracleUserPass",
+            "DatabricksHmsCredential",
+            "SapS4HanaSapS4HanaCredential",
+            "SapS4HanaSapS4HanaUserPass",
+            "SapEccSapEccCredential",
+            "SapEccSapEccUserPass",
+            "SnowflakeCredential",
+            "PowerBICredential",
+            "PowerBIDelegated",
+            "PowerBIFpa",
+            "PowerBIMsi",
+            "FabricCredential",
+            "FabricDelegated",
+            "FabricFpa",
+            "FabricMsi",
+            "TridentCredential",
+            "TridentDelegated",
+            "TridentMsi",
+            "DataverseCredential",
+            "DataverseMsi"
+          ],
+          "type": "string",
+          "description": "The properties of scan, vary by datasource type.",
+          "x-ms-enum": {
+            "name": "ScanAuthorizationType",
+            "modelAsString": true
+          }
+        },
+        "scanResults": {
+          "description": "Scan results of this scan.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScanResult"
+          },
+          "readOnly": true
+        },
+        "scanResultsWithIngestion": {
```

Comment:
What's the difference between scanResults and scanResultsWithIngestion? Are both of these really needed?

Reply:
scanResultsWithIngestion is the new model that substitute the scanResults. I just removed the latter one in the code.

Reply:
The key of this field is scanResults, which is designed to keep with previous versions. And the model within should be updated one, which is scanResultsWithIngestion